src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj

```diff
@@ -305,4 +305,12 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
+  <Target Name="AfterBuild">
+    <!-- Drop a file in the artifacts directory containing the ANCM version number -->
+    <ItemGroup>
+      <VersionFileContents Include="$(AspNetCoreModuleVersionMajor).$(AspNetCoreMinorVersion).$(AssemblyBuild).$(AspNetCorePatchVersion)" />
```

Comment:
I'm going to want to verify these numbers tomorrow. I don't believe these are correct, but want to confirm before I correct you.